devtron-labs/dashboard
# Fix - External link is not generated properly when same key is used multiple times

# Description

These changes fix the issue when any key ({namespace}, {containerName}, etc. ) in an external link is used two or more times causing the URL/link to be not generated.

Fixes # https://github.com/devtron-labs/devtron/issues/1752

## Type of change


# How Has This Been Tested?

Tests have been performed manually,

1. Go to Global configurations
2. Go to the External Link section
3. Add new link as - https://devtron.info/grafana/{containerName}/test/{containerName}
4. Go to the AppDetails page
5. Observe the app level external link section

**Expected behavior**

It should generate the proper link even when the same key is used multiple times.


# Checklist:
